munkitools/munkitools6.munki.recipe

```diff
@@ -31,6 +33,10 @@ MUNKI_ICON should be overridden with your icon name.
     <string>com.github.autopkg.munki.munkitools6</string>
     <key>Input</key>
     <dict>
+        <key>GITHUB_REPO</key>
+        <string>munki/munki</string>
+        <key>DISABLE_CODE_SIGNATURE_VERIFICATION</key>
+        <string>yes</string>
```

Comment:
(Then for a recipe override, when downloading the signed package, you'd override the value to `<false/>`)

Reply:
For more context:
https://github.com/autopkg/autopkg/blob/master/Code/autopkglib/CodeSignatureVerifier.py#L301

```python
        if self.env.get("DISABLE_CODE_SIGNATURE_VERIFICATION"):
```

if Python evaluates self.env.get("DISABLE_CODE_SIGNATURE_VERIFICATION") as "truthy", then it skips the verification. From Python's perspective, "truthy" might mean:
Boolean True
Non-empty string
Non-empty array or dictionary
Non-zero number
(there may be others)

"falsey" would mean:
Boolean False
Empty string
Empty array or dictionary
0
None

At the command-line, since everything is a string, the convention is to use "" for False, and a non-empty string like "Yes" or "True" for True. ("False" and "No"  are also "truthy")

But in a recipe, where we _can_ use types other than strings, let's use Booleans here.

Reply:
Thanks for the extra details, I made those changes in the latest commit.